docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -74,10 +87,55 @@ Claims, and to run MongoDB in the new cluster:
 2. :ref:`Add Persistent Volume Claims <Step 11: Create Kubernetes Persistent Volume Claims>`.
 3. :ref:`Create the Config Map <Step 3: Configure Your BigchainDB Node>`.
-4. :ref:`Run MongoDB instance <Step 12: Start a Kubernetes StatefulSet for MongoDB>`.
+4. :ref:`Prepare the Kubernetes Secrets <Step 3: Configure Your BigchainDB Node>`, as per your
+   requirement i.e. if you do not want a certain functionality, just remove it from the
+   ``configuration/secret.yaml``.
+5. :ref:`Run MongoDB instance <Step 12: Start a Kubernetes StatefulSet for MongoDB>`.
+
+
+Step 3: Start NGINX service, Assign DNS to NGINX Pubic IP and run NGINX deployment
+----------------------------------------------------------------------------------
+
+Please see the following pages:
+
+* :ref:`Start NGINX service <Step 4: Start the NGINX Service>`.
+* :ref:`Assign DNS to NGINX Public IP <Step 5: Assign DNS Name to the NGINX Public IP>`.
+* :ref:`Run NGINX deployment <Step 9: Start the NGINX Kubernetes Deployment>`.
+
+
+Step 4: Verify network connectivity between the MongoDB instances
+-----------------------------------------------------------------
+
+Make sure your MongoDB instances can access each other over the network. *If* you are deploying
+the new MongoDB node in a different cluster or geographical location using Azure Kubernetes container
+service, you will have to setup networking (name resolution) between the two clusters using `Kubernetes
+Services <https://kubernetes.io/docs/concepts/services-networking/service/>`_.
+
+Assuming we have an existing MongoDB instance ``mdb-instance-0`` residing in Azure data center location ``westeurope`` and we
+want to add a new MongoDB instance ``mdb-instance-1`` located in Azure data center location ``eastus`` to the existing MongoDB
+replica set. Unless you already have explicitly setup networking for ``mdb-instance-0`` to communicate with ``mdb-instance-1`` and
+vice versa, we will have to add a Kubernetes service in each cluster to accomplish this goal in order to setup a
+MongoDB replica set.
+
+* This configuration is located in the file ``mongodb/mongo-conn-svc``.
 
+* Set the name of the ``metadata.name`` to the host name of the MongoDB instance you are trying to connect to.
+  For instance if you are configuring this service on cluster with `mdb-instance-0` then the ``metadata.name`` will
+  be ``mdb-instance-1`` and vice versa.
 
-Step 3: Add the New MongoDB Instance to the Existing Replica Set
+* Set ``spec.ports.port[0]`` to the ``mongodb-backend-port`` from the ConfigMap.
+
+* Set ``spec.externalName`` to the FQDN mapped to NGINX Public IP of the cluster you are trying to connect to.
+  For more information about the FQDN please refer to: :ref:`Assign DNS Name to the NGINX Public
+  IP <Step 5: Assign DNS Name to the NGINX Public IP>`
+
+.. note::
+
+   We will have to perform this operation repeatedly across clusters (if any) for all
+   the MongoDB instances to be able to talk to each other.
```

Comment:
@ttmc that is true. This will get a bit messy when you have a large number of clusters in different zones.
I can add something like this:

> This operation needs to be replicated `n-1` times per node for a `n` node cluster, with different FQDNs for each node.

@krish7919 does it make sense?